cmd/cli/uninstall_mesh.go

```diff
@@ -199,6 +182,63 @@ func (d *uninstallMeshCmd) run() error {
 			fmt.Fprintf(d.out, "OSM namespace CANNOT be deleted in a managed environment\n")
 		}
 	}
+
+	err := d.deleteClusterResources()
+	return err
+}
+
+func (d *uninstallMeshCmd) findFlagSpecifiedMesh(meshInfoList []meshInfo) (bool, bool) {
```

Comment:
how about we hide the double return value, to return a single boolean on whether the mesh was found via the specified info.